### DISCUSSION TITLE
Did everyone upgrade v1.16.0 smoothly?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#8921] [Build and Sync] [answered]
Did everyone upgrade v1.16.0 smoothly? I have to upgrade to v.15.2 because of the abnormal upgrade!

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! 

The upgrade has gone smoothly here at least. Did you meet any issues when upgrading to v1.16.0?

**Comment by ChaiWenyu1983** [comment]
"TLS" error will be reported in the middle. There is no screen capture at that time. Can you share your upgrade command.


My are as follows:


git pull


git checkout v1.16.0;


git submodule update


make clean








&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月27日(星期一) 下午5:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] Did everyone upgrade v1.16.0 smoothly? (Discussion #8921)





 
Hey @ChaiWenyu1983!
 
The upgrade here has gone smoothly here at least. Did you meet any issues when upgrading to v1.16.0?

**Reply by rjan90** [reply]
First, make sure that you are on a compatitble go-version. You should be on a Go-version of v1.17.9 or higher. Check your version with: `go version`

Then I just follow the regular upgrade-in-place commands: https://lotus.filecoin.io/storage-providers/operate/upgrades/#upgrade-in-place.

```
export RUSTFLAGS="-C target-cpu=native -g"
export FFI_BUILD_FROM_SOURCE=1
export FFI_USE_CUDA=1 #if you are using CUDA 
git pull
git checkout v1.16.0
git submodule update
make clean
make all
(sudo) make install
```

**Comment by ChaiWenyu1983** [comment]
The upgrade failed. After installing "1.15.2", the "git.518dc962e.dirty" appears. How to remove this ".Dirty"?

**Reply by rjan90** [reply]
Hey @ChaiWenyu1983! You can follow the process here to get a clean Lotus install: https://github.com/filecoin-project/lotus/discussions/8962#discussioncomment-3071983
